django_etuovi/etuovi.py

```diff
@@ -33,9 +34,11 @@ def get_session() -> FTP:
     )
 
 
-def create_xml_file(items: List[BaseClass]) -> str:
+def create_xml_file(items: List[BaseClass], file_path: str = "") -> str:
     element_tree = ElementTree(create_element_tree(items))
     filename = get_filename()
+    if file_path:
+        filename = path.join(file_path, filename)
```

Comment:
See my comment [here](https://github.com/City-of-Helsinki/django-oikotie/pull/17#discussion_r634080083).